packages/ag-charts-community/src/api/preset/priceVolumePreset.ts

```diff
@@ -313,6 +314,12 @@ function createPriceSeries(
                     fill: fill ?? PALETTE_DOWN_STROKE,
                     stroke: stoke ?? PALETTE_DOWN_STROKE,
                 } satisfies AgRangeAreaSeriesOptions,
+                {
+                    type: 'line',
+                    ...singleKeys,
+                    stroke: fromTheme(theme, (t) => t.overrides?.line?.series?.stroke) ?? DEFAULT_STROKES.GRAY,
```

Comment:
Could we use `PALETTE_NEUTRAL_STROKE` here for consistency? I'd rather we didn't hardcode these colours.

Reply:
Ok but that's blue, jira says it should be gray is that ok?

Reply:
Discussed offline, let's roll with the original proposal